File: gestion/src/main/java/com/hps/gestion/dto/Credentials.java
package com.hps.gestion.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class Credentials {
    private String jiraURL;
    private String jiraUsername;
    private String jiraPassword;
    private String bitbucketURL;
    private String bitbucketUsername;
    private String bitbucketPassword;
}

File: gestionUI/src/app/shared/credentials.ts
export class Credentials{
    jiraURL : string;
    jiraUsername : string;
    jiraPassword : string;
    bitbucketURL : string;
    bitbucketUsername : string;
    bitbucketPassword : string;
    constructor(jiraURL : string ,jiraUsername : string , jiraPassword : string ,bitbucketURL : string, bitbucketUsername : string , bitbucketPassword : string){
        this.jiraURL = jiraURL;
        this.jiraUsername = jiraUsername;
        this.jiraPassword = jiraPassword;
        this.bitbucketURL = bitbucketURL;
        this.bitbucketUsername =bitbucketUsername;
        this.bitbucketPassword = bitbucketPassword;
    }
}